github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -347,8 +347,8 @@ presubmits:
     annotations:
       fork-per-release: "true"
       testgrid-dashboards: kubevirt-containerized-data-importer-presubmits
-    always_run: false
-    optional: true
+    always_run: true
+    optional: false
```

Comment:
Should we wait until we know why this lane is much slower than non-wffc?

Reply:
I don't think so - we want this to gate PRs so we are confident no WFFC regressions make their way in